test/constructors.jl

```diff
@@ -60,7 +60,7 @@ module TestConstructors
     @test isa(Z, NullableVector{Int})
     @test Z.isnull == [false, true, false, false, false, false, true]
 
-    Y = NullableArray([1, "na", 2, 3, 4, 5, "na"], Int, ASCIIString)
+    Y = NullableArray([1, "na", 2, 3, 4, 5, "na"], Int, Compat.ASCIIString)
```

Comment:
I'd rather change this into `Y = NullableArray([1, nothing, 2, 3, 4, 5, nothing], Int, Void)` or `Y = NullableArray(Nullable{Int}[1, Nullable(), 2, 3, 4, 5, Nullable()])`.